modules/ossm-auto-route.adoc

```diff
@@ -26,7 +26,7 @@ spec:
     - bookinfo.example.com
```

Comment:
This file is missing a content type attribute. Please verify that all the files in this PR have content type attributes and add any that might be missing.

Reply:
@rolfedh I've updated the files that were missing a content type attribute so that they now have one.